### ISSUE TITLE
pci passthrough possible for smbus?

### ISSUE THREAD

**Issue by squont** [dockur/windows#239]
### tldr
can I pass through pci devices such as the SMBus to the guest? If so how? I am getting errors trying to do that(`failed to open /dev/vfio/5: No such file or directory`). 

PS: Thanks a bunch for this container - so very convenient! Heard about it from my spirit guide, Wolfgang

### goal
unsure if possible. To run bios update software that is _wndows only_ in a kvm windows guest with archlinux host. 

I've tried with, and without adding `/dev/vfio` to the devices section in the compose file and it didn't seem to make any difference. Not sure that it should or not. Just trying things. 

### environment
- Archlinux
- z1 extreme apu
- amd_iommu=on param set

```ini
lspci | grep SMBus
00:14.0 SMBus: Advanced Micro Devices, Inc. [AMD] FCH SMBus Controller (rev 71)
```
```ini
 sudo dmesg | grep -i group\ 5
[    0.815400] pci 0000:00:14.0: Adding to iommu group 5
[    0.815436] pci 0000:00:14.3: Adding to iommu group 5
```

```ini
sudo dmesg | grep IOMMUv2
[    0.878446] AMD-Vi: AMD IOMMUv2 loaded and initialized
```
``` docker-compose.yml
version: "3"
services:
  windows:
    image: dockurr/windows
    container_name: windows
    environment:
      ARGUMENTS: "-device vfio-pci,host=00:14.0 -device usb-host,vendorid=0x17ef,productid=0x6182"
    devices:
      - /dev/kvm
      - /dev/bus/usb
    cap_add:
      - NET_ADMIN
    ports:
      - 8006:8006
      - 3389:3389/tcp
      - 3389:3389/udp
    stop_grace_period: 2m
    restart: on-failure
    volumes:
      - ./data:/storage
```

### Compose log / error log
```ini
[+] Running 2/0
 ✔ Network windows_default  Created                                                                                  0.0s
 ✔ Container windows        Created                                                                                  0.0s
Attaching to windows
windows  | ❯ Starting Windows for Docker v2.04...
windows  | ❯ For support visit https://github.com/dockur/windows
windows  |
windows  | ❯ Booting Windows using QEMU emulator version 8.2.1 ...
windows  | ❯ ERROR: qemu-system-x86_64: -device vfio-pci,host=00:14.0: vfio 0000:00:14.0: failed to open /dev/vfio/5: No such file or directory

```
